micro_manager/micro_manager.py

```diff
@@ -71,8 +71,6 @@ def __init__(self, config_file: str) -> None:
             self._rank,
             self._size)
 
-        micro_file_name = self._config.get_micro_file_name()
-
```

Comment:
Is this related to crash handling?

Reply:
No this is a formatting error. I will revert this and open a separate pull request with changes.